tests/components/dataset_activity/test_views.py

```diff
@@ -63,6 +66,16 @@ async def test_list_dataset_activity_returns_list_of_existing_item_and_dataset_a
 
         assert received_total == 6
 
+        file_activity_logs = body('.result[] | select(.index=="file")').all()
```

Comment:
Please make everything consistent in naming, since we use `item_activity` so it's better to keep it like that to avoid confusion.
```suggestion
        item_activity_logs = body('.result[] | select(.index=="file")').all()
```